### PR TITLE
Add test to storyviews if targetElement is null

### LINKED ISSUE #5772: [BUG] RSOE if the $:/tags/SideBar is modified
**Describe the bug**
`Uncaught TypeError: Cannot read property 'nodeType' of null` is invoked by modifying the `$:/tags/SideBar` tiddler in any(?) way.

**To Reproduce**
1. Go to https://tiddlywiki.com/prerelease/#%24%3A%2Ftags%2FSideBar
2. Click on edit
3. Modify the tiddler in any way, e.g: type sth in the text field, or add a tag, or change the list field
4. RSOE `Uncaught TypeError: Cannot read property 'nodeType' of null`

### PR DESCRIPTION
**Pull request by BurningTreeC**
This PR tests if the `targetElement` is `null` in all of the three storyviews before testing for `targetElement.nodeType` which would cause a RSOD if `targetElement` is `null`

Fixes #5772

### CODE REVIEW

File: core/modules/storyviews/classic.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ ClassicStoryView.prototype.navigateTo = function(historyInfo) {
 	var listItemWidget = this.listWidget.children[listElementIndex],
 		targetElement = listItemWidget.findFirstDomNode();
 	// Abandon if the list entry isn't a DOM element (it might be a text node)
-	if(targetElement === undefined || targetElement.nodeType === Node.TEXT_NODE) {
+	if(!targetElement || targetElement === undefined || targetElement.nodeType === Node.TEXT_NODE) {
```
Comment (saqimtiaz) [inline core/modules/storyviews/classic.js:30]: I think you can simplify this to:
`if(!targetElement || targetElement.nodeType === Node.TEXT_NODE) {`

Comment (BurningTreeC) [reply core/modules/storyviews/classic.js:30]: thanks, I'll do so @saqimtiaz

**Comment by pmario** [comment]
What did you do, to get the RSOD?

I'm testing "Server Sent Events" atm, which causes a RSOD because of the same issue.

**Comment by Alzacon** [comment]
I was going to report the RSOD that (i think that) this PR solves it.

@pmario , I found it when I try to edit something in any of ¿build-in? tiddlers, you can test it in https://tiddlywiki.com/prerelease/ and you can try to edit any tiddlers that are displayed.

**Comment by Jermolene** [comment]
Thanks @BurningTreeC @saqimtiaz @pmario @Alzacon, apologies for the delay in merging.
